File: setup/jieba.ts
/* eslint-disable no-console, @typescript-eslint/no-unsafe-return */
const execSync = require('child_process').execSync

console.log('jieba')
execSync('mkdir -p build/resource/ooooevan-jieba/dict')
execSync('cp node_modules/ooooevan-jieba/dict/* build/resource/ooooevan-jieba/dict')

export {}

File: setup/jieba.py
#!/usr/bin/env python3

import os, glob, shutil

print('copying jieba assets')
os.makedirs('build/content/resource/ooooevan-jieba/dict', exist_ok=True)
for asset in glob.glob('node_modules/ooooevan-jieba/dict/*'):
  print(' ', os.path.basename(asset))
  shutil.copy(asset, os.path.join('build/content/resource/ooooevan-jieba/dict', os.path.basename(asset)))
